x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SLMSnapshotBlockingIntegTests.java

```diff
@@ -268,7 +284,9 @@ public void testBasicFailureRetention() throws Exception {
                     assertEquals(ClusterHealthStatus.RED, client().admin().cluster().prepareHealth().get().getStatus()),
                 30, TimeUnit.SECONDS);
 
-            final String masterNode = blockMasterFromFinalizingSnapshotOnIndexFile(REPO);
+            final String masterNode;
+            masterNode = blockMasterFromFinalizingSnapshotOnIndexFile(REPO);
```

Comment:
What was this change for?

Reply:
I briefly experimented with only blocking data nodes to induce `PARTIAL` snapshots, but it didn't work out - this was an artifact of that, I guess I missed cleaning it up. Put it back now.